specification/cognitiveservices/data-plane/Speech/SpeakerRecognition/identification/identification-swagger.json

```diff
@@ -0,0 +1,771 @@
+{
+  "swagger": "2.0",
+  "info": {
```

Comment:
@HeidiHanZhang -- Would it be possible to add ```description``` here? It's not necessary, but a good practice to have it.

Reply:
done